### PR TITLE
Adding new CQ export svg features

### PR DESCRIPTION
**Pull request by Shimwell**
## Proposed changes

Adds improved svg export options such as width and height

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Code refactoring
- [ ] Documentation Update (if none of the other choices apply)
- [ ] New tests

## Checklist

- [ ] Pep8 applied
- [ ] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Shimwell** [comment]
great looks like the ci is working again (I updated the docker image)

**Comment by Shimwell** [comment]
This was failing the circle ci as that docker image has the older version of ocp on it. 

https://github.com/CadQuery/CQ-editor/issues/222

However this has been fixed now and it the docker image used by circle ci has also been rebuilt

File: paramak/parametric_neutronics/make_faceteted_neutronics_model.py [hunk 1]
```diff
@@ -109,9 +109,9 @@ def find_all_surfaces_of_reflecting_wedge(new_vols):
         #area = surface.area()
         vertex_in_surface = cubit.parse_cubit_list("vertex", " in surface " + str(surface_id))
         if surface.is_planar() == True and len(vertex_in_surface) == 4:
-        surface_info_dict[surface_id] = {'reflector': True}
+            surface_info_dict[surface_id] = {'reflector': True}
```
Comment (RemDelaporteMathurin) [inline paramak/parametric_neutronics/make_faceteted_neutronics_model.py:112]: This line isn't tested

File: paramak/parametric_neutronics/make_faceteted_neutronics_model.py [hunk 2]
```diff
@@ -109,9 +109,9 @@ def find_all_surfaces_of_reflecting_wedge(new_vols):
...
         else:
-        surface_info_dict[surface_id] = {'reflector': False}
+            surface_info_dict[surface_id] = {'reflector': False}
```
Comment (RemDelaporteMathurin) [inline paramak/parametric_neutronics/make_faceteted_neutronics_model.py:114]: This line isn't tested

File: paramak/parametric_neutronics/make_faceteted_neutronics_model.py (hunk 1 above)
Comment (Shimwell) [reply paramak/parametric_neutronics/make_faceteted_neutronics_model.py:112]: yep this is one of those lines that we can't test without trelis being on the docker image, sorry

File: paramak/parametric_neutronics/make_faceteted_neutronics_model.py (hunk 2 above)
Comment (Shimwell) [reply paramak/parametric_neutronics/make_faceteted_neutronics_model.py:114]: same here, we will add trelis as soon as possible but it isn't possible yet

**Comment by Shimwell** [comment]
I think this is ready to merge then I shall start work on updating the stp and stl exports

**Pull request merged**